src/translation.py

```diff
@@ -7,17 +7,20 @@ class Translation:
     def __init__(self, reference: Sentence = None, hypothesis: Sentence = None):
         self.r = reference
         self.h = hypothesis
+        self.errors = None
 
     def count_all_lemmas(self):
         lemmas = Counter(
-            [word.lemma for word in self.r.words if word.lemma != "_"] +
-            [word.lemma for word in self.h.words if word.lemma != "_"]
+            [word.lemma for word in self.r.words if word.lemma] +
```

Comment:
this if is for the case where it is None or "", if you meant the first the second may cause bugs (from self experience...)